### ISSUE TITLE
Add information about running Geoclimate with bounding box

### ISSUE THREAD

**Issue by j3r3m1** [orbisgis/geoclimate#603]
We do not have example in the doc. Maybe we could at least add how to do that in the description of the [config file](https://github.com/orbisgis/geoclimate/wiki/Default-case-with-OSM-%5BL%5D#understand-the-configuration-file) (for the input keyword).

**Comment by franetibe** [comment]
I can take care of this.

I propose to add a new tutorial page in the wiki below the default cases, what do you think ?

**Comment by j3r3m1** [comment]
OK for that but then only a short version telling what to change from the default case ?

**Comment by franetibe** [comment]
Yes, just to explain what the user needs to change compared to the default case

**Comment by 4estone** [comment]
Just add a configuration file example with bbox parameter. It sould do the trick...

**Comment by ebocher** [comment]
Feel free to submit to proposal to the wiki.

**Comment by ebocher** [comment]
Under the pipe by @franetibe

**Comment by 4estone** [comment]
Thanks
